jablib/src/main/java/org/jabref/model/entry/field/StandardField.java

```diff
@@ -138,8 +138,10 @@ public enum StandardField implements Field {
     OWNER("owner"),
     TIMESTAMP("timestamp", FieldProperty.DATE),
     CREATIONDATE("creationdate", FieldProperty.DATE),
+    CITATIONCOUNT("Citation count", FieldProperty.CITATION_COUNT),
```

Comment:
Remove the field property. (second parameter)

Reply:
done